app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -815,11 +580,11 @@ impl Area {
                         });
                 }
 
-                let index  = node.payload.index;
+                let index = node.payload.index;
                 let length = node.payload.length;
-                let label  = model.label.clone_ref();
+                let label = self.label.clone_ref();
```

Comment:
I dont like setting FRP from model. we are not doing it anywhere. FPR should be set in Area for model. This is a pattern we are using everywhere. I'd like to keep this pattern. When we will be transitioning to the next FRP implementation this will be required. Can we keep the pattern of setting FRP network in entity that MANAGES model, as it was before, please? If not, lets have a call, please.

Reply:
We're not extending FRP of the Area here, we're extending `port_network` instead.